cloud-formation/membership-app.cf.yaml

```diff
@@ -233,7 +242,7 @@ Resources:
   InstanceSecurityGroup:
     Type: AWS::EC2::SecurityGroup
     Properties:
-      GroupDescription: Open up ssh access and enable HTTP access on the configured
+      GroupDescription: Enable HTTP access on the configured
```

Comment:
Could revert this description change now that we're not removing the port 22 access?